docker-compose.yml

```diff
@@ -24,7 +24,6 @@ services:
       FIDES__DATABASE__SERVER: "fides-db"
       FIDES__DEV_MODE: "True"
       FIDES__EXECUTION__WORKER_ENABLED: "True"
-      FIDES_TEST_MODE: "True"
```

Comment:
The quickstart can work with this set to True also, the important detail is that the same value is set on the worker node too, because it effects which database URL the container will try to connect to.